pom.xml

```diff
@@ -147,6 +147,12 @@
 
       <dependency>
         <groupId>com.facebook.nifty</groupId>
+        <artifactId>nifty-ssl</artifactId>
```

Comment:
I suppose this has to go to a separate package (e.g. swift-ssl) for the same reason nifty-ssl was split from nifty. Since most people use nifty through swift, separating only nifty makes little sense.